src/game.rs

```diff
@@ -143,15 +181,25 @@ impl Game {
 
         self.shooter.render(&mut canvas);
 
-        for bullet in self.bullets.iter_mut() {
+        for bullet in self.shooter_bullets.iter_mut() {
+            bullet.render(&mut canvas);
+        }
+
+        for bullet in self.alien_bullets.iter_mut() {
             bullet.render(&mut canvas);
         }
 
         for alien in self.alien_group.aliens.iter_mut() {
             alien.render(&mut canvas);
         }
 
-        self.score_board.render(&mut canvas)
+        self.score_board.render(&mut canvas);
```

Comment:
Don't like that we still render evertyhing else anyways when the shooter is dead.